BUILD.bazel

```diff
@@ -7482,6 +7482,14 @@ config_setting(
     },
 )
 
+config_setting(
+    name = "macos_arm64",
+    values = {
+        "apple_platform_type": "macos",
+        "cpu": "darwin_arm64",
+    },
+)
+
```

Comment:
Should be grouped with other macos configs

Reply:
regrouped with other macos configs.